.gitignore

```
@@ -1,4 +1,5 @@
 cluster-up/
+.kubevirtci
```

tiny - just for better readability, please add `/` so it will be clear that this is a directory (and not the kubevirtci script, for instance).

```suggestion
.kubevirtci/
```